cpp/arcticdb/processing/aggregation.cpp

```diff
@@ -48,7 +98,7 @@ std::optional<DataType> Sum::finalize(SegmentInMemory& seg, bool, size_t unique_
             auto col = std::make_shared<Column>(make_scalar_type(that->data_type_), unique_values, true, false);
             memcpy(col->ptr(), that->aggregated_.data(), that->aggregated_.size());
             seg.add_column(scalar_field_proto(that->data_type_, that->get_output_column_name().value), col);
-            col->set_row_data(unique_values);
+            col->set_row_data(unique_values - 1);
```

Comment:
These changes do not affect the aggregation behaviour, but means that the `SegmentInMemory` row count is accurate.